react/week3/react-Week-Three-HW/todo-app/src/Components/Todo.js

```diff
@@ -0,0 +1,196 @@
+import React,{Component} from "react";
+import PropTypes from "prop-types";
+
+export const RenderList = (props) => {
+	const editstate = props.todos.editstate;
+
+	const details = editstate ? (
+		<input
+			type="text"
+			placeholder="update todo"
+			onChange={(event) => {
+				const updatedEvent = event.target.value;
+
+				return props.updatedTodo(updatedEvent);
+			}}
+		/>
+	) : (
+		` ${props.todos.description}  ||  ${props.todos.deadline}`
+	);
```

Comment:
This is hard to follow. Im not entirely sure what you are doing here or what this does. 

Try and simplify it maybe with a function or better variables. 

Like what is details? Is there an editstate for all todoitems, seems weird, Why are you returning the props in the onChange event. These are the things that confuse me